### DISCUSSION TITLE
The List packages for an organization(/orgs/{org}/packages) endpoint does not return a Link response header for pagination

### DISCUSSION THREAD

**Discussion by mjfpalmer** [community/community#128552] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

The `/orgs/{org}/packages` ([List packages for an organization](https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28#list-packages-for-an-organization)) endpoint accepts `page` and `per_page` query parameters, and should return a `Link` response header, but it doesn't, even when there is another page. I've had to construct the next page Uri manually for this endpoint.

**Comment by timursaikaliev** [comment]
I've run into this problem too.
I noticed that if you specify the **package_type** as _"container"_ `/orgs/{org}/packages?package_type=container` the `Link` header is there, but if you specify the **package_type** as _"npm"_ or "_nuget"_, there is no `Link` header.

**Reply by denniseffing** [reply]
Same here.
In addition, we noticed that the endpoint [List package versions for a package owned by an organization](https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28#list-package-versions-for-a-package-owned-by-an-organization) actually **does** return the `link` header.

Only the [List packages for an organization](https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28#list-packages-for-an-organization) endpoint does not return the `link` header.
